components/handlers/module_edit.py

```diff
@@ -45,6 +45,11 @@ def POST(self, *test_data):
             module_name = input_data.name
             module_desc = input_data.desc
             module_mc = input_data.mc
+
+            #validate inputs
+
+            if not check_string.check_name(module_name):
+                return check_string.outcome_invalid()
         except AttributeError:
             return Outcome().POST("edit_module", False, module_code)
 
```

Comment:
Can replace line 56 to 62 with check_mc instead (if they are checking for the same thing)